core/src/main/java/io/crate/types/ObjectType.java

```diff
@@ -56,9 +59,21 @@ public String getName() {
     @Override
     @SuppressWarnings("unchecked")
     public Map<String, Object> value(Object value) {
+        if (value instanceof BytesRef) {
```

Comment:
do we need a string check too? like e.g in https://github.com/crate/crate/blob/50b1876124e74031063a7cac1c1cc52acbedc7ce/core/src/main/java/io/crate/types/BooleanType.java#L70

not sure where we still have the string case though, if there is some we need a test